### PR TITLE
New launch file for both webcam & talker

### PR DESCRIPTION
**Pull request by pablisho**

### CODE REVIEW

File: applications/launch/proxy.launch [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<launch>
+  <arg name="urlServer" />
```
Comment (adamantivm) [inline applications/launch/proxy.launch]: @pablisho could you please just use 'url' as a param? Or if you choose a different name, please use this_fromat, not lowerCamelCase. Thanks!

Comment (pablisho) [inline applications/launch/proxy.launch]: @adamantivm Done! Changed to server_url

File: applications/launch/proxy.launch [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<launch>
+  <arg name="server_url" />
+  <arg name="video" default="/dev/video0"
```
Comment (adamantivm) [inline applications/launch/proxy.launch]: there's a missing closing bracket here!

Comment (pablisho) [inline applications/launch/proxy.launch]: @adamantivm Thank you! Fixed!

**Pull request merged**